stl/inc/experimental/random

```diff
@@ -0,0 +1,52 @@
+// random experimental header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _EXPERIMENTAL_RANDOM_
+#define _EXPERIMENTAL_RANDOM_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#include <random>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace experimental {
+    inline namespace fundamentals_v3 {
+
+        inline default_random_engine& _Engine() {
+            thread_local default_random_engine _Eng{random_device{}()};
+            return _Eng;
+        }
+
+        template <class _IntType>
+        _IntType randint(const _IntType _Min, const _IntType _Max) {
+            static_assert(_Is_any_of_v<remove_cv_t<_IntType>, short, int, long, long long, unsigned short, unsigned int,

```

Comment:
I believe you could use is_integral_v here. Also the message should read “integral Type“ at the end rather the implementation detail IntType

Reply:
I don't think I can, is_integral_v includes the char types which isn't what is listed in the requirements
https://eel.is/c++draft/rand.req.genl#1.5

Reply:
Just realised there's already a macro for this.